recipes/shared-memory-dict/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "shared-memory-dict" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shared-memory-dict-{{ version }}.tar.gz
+  sha256: 8294257e1a2466afb5e19a878a8c446325b8913d57d6d9578ccba905564a974d
+
+build:
+  number: 0
+
+outputs:
+  - name: shared-memory-dict
+    build:
+      noarch: python
+    script: build_base.sh
+    requirements:
+      host:
+        - pip
+        - python >=3.6,<4.0
+      run:
+        - python >=3.6,<4.0
+    test:
+      imports:
+        - shared_memory_dict
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  - name: shared-memory-dict-aiocache
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage('shared-memory-dict', exact=True) }}
+        - aiocache >=0.11.1,<0.12.0
+    test:
+      imports:
+        - shared_memory_dict
+
+  - name: shared-memory-dict-all
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage('shared-memory-dict', exact=True) }}
+        - aiocache >=0.11.1,<0.12.0
+        - django >=3.0.8,<4.0.0
+    test:
+      imports:
+        - shared_memory_dict
+
+  - name: shared-memory-dict-django
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage('shared-memory-dict', exact=True) }}
+        - django >=3.0.8,<4.0.0
+    test:
+      imports:
+        - shared_memory_dict
+
+about:
+  home: https://github.com/luizalabs/shared-memory-dict
+  summary: A very simple shared memory dict implementation
+  license: MIT
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - maxb2
```

```suggestion
    - maxb2
  feedstock-name: shared-memory-dict
```
Unrelated to the error, this just makes sure that the feedstock name is correct.